src/TuDa.CIMS.Web/Components/AssetList.razor

```diff
@@ -0,0 +1,195 @@
+<h3>Bestandsliste</h3>
+
+@code {
+
+// Parameter Items for asset list
+    [Parameter] public string[] Items { get; set; }
```

Comment:
🔧 For a parameter we should add a doc comment like
```suggestion
    /// <summary>
    /// List of items to be shown
    /// </summary>
    [Parameter] public string[] Items { get; set; }
```

Reply:
So when you want to use the component you can see some information about the parameters